repositories/memory/topics/get_topic_test.go

```diff
@@ -0,0 +1,98 @@
+package topics
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/matheusmosca/walrus/domain/entities"
+	"github.com/matheusmosca/walrus/domain/vos"
+	"github.com/stretchr/testify/require"
+)
+
+type args struct {
+	topicName vos.TopicName
+}
+type testCase struct {
+	description string
+	args        args
+	beforeRun   func(storage map[vos.TopicName]entities.Topic)
+	wantErr     error
+}
+
+func TestGetTopic_Success(t *testing.T) {
+	// positiveTestcase creates a topic and stores it in the repository.storage as a part of its beforeRun routine
+	// hence these test cases should run with positive results for GetTopic method
+	positiveTestcase := []testCase{
```

Comment:
```suggestion
	tests := []testCase{
```

Reply:
Ack. Replaced the positiveTestcase with tests